### ISSUE TITLE
Environment variables not loading into gatsy.config

### ISSUE THREAD

**Issue by catonmat** [jbolda/gatsby-source-airtable#203]
The following code works and I can load my AirTable data into GraphQL queries within my site. 

```js
{
      resolve: `gatsby-source-airtable`,
      options: {
        apiKey: `keyxxxxxxxxxxx`, // may instead specify via env, see below
        concurrency: 5, // default, see using markdown and attachments for more information
        tables: [
          {
            baseId: `appxxxxxxxxxxx`,
            tableName: `table`,
            separateNodeType: false, // boolean, default is false, see the documentation on naming conflicts for more information
            separateMapType: false, // boolean, default is false, see the documentation on using markdown and attachments for more information
          }
        ]
      }
    },
```

However, when I follow your instructions in the README.md and use environment variables instead, I get the following error: `The gatsby-source-airtable plugin has generated no Gatsby nodes. Do you need it?`

```js
{
      resolve: `gatsby-source-airtable`,
      options: {
        apiKey: process.env.AIRTABLE_API_KEY, // may instead specify via env, see below
        concurrency: 5, // default, see using markdown and attachments for more information
        tables: [
          {
            baseId: process.env.AIRTABLE_BASE,
            tableName: `table`,
            separateNodeType: false, // boolean, default is false, see the documentation on naming conflicts for more information
            separateMapType: false, // boolean, default is false, see the documentation on using markdown and attachments for more information
          }
        ]
      }
    },
```

I have my ENV variables inside `.env.development` in the root of the project and am testing my graphQl queries in GraphiQL in the `gatsby develop` environment.

I have also restarted my dev servef and re-ran `npm install` many times to eliminate the possibility of this being a dependency related issue.

**Comment by jbolda** [comment]
👋 

Are you certain that you have (I assume you are using) `dotenv` wired up correctly? It may be worth `console.log` the envs in your gatsby config to be absolutely certain it is at least making it that far.

**Comment by catonmat** [comment]
Hi @jbolda , 

I have successfully installed `dotenv` and placed it at the top of my `gatsby-config.js`. When I `console.log(process.env)` inside my `gatsby-config.js` I see many different ENV variables. 

Also, when I run `AIRTABLE_API_KEY=xxxx gatsby develop` the api key is loaded into config.

However, when I run `gatsby develop` with my api key in `.env.development`, this data is not loaded into the config.

I've attached more info below that may be useful for debugging the issue:
- Node v.14.0.1
- NVM v0.35.3
- Gatsby version: 2.21.13
- Gatsby CLI version: 2.12.10
- node arch: x64
- node platform: darwin
- node process versions: 
```
{
  node: '14.1.0',
  v8: '8.1.307.31-node.32',
  uv: '1.37.0',
  zlib: '1.2.11',
  brotli: '1.0.7',
  ares: '1.16.0',
  modules: '83',
  nghttp2: '1.40.0',
  napi: '6',
  llhttp: '2.0.4',
  openssl: '1.1.1g',
  cldr: '36.1',
  icu: '66.1',
  tz: '2019c',
  unicode: '13.0'
}
```
